### DISCUSSION TITLE
Target SDK 34 bonding is not working

### DISCUSSION THREAD

**Discussion by hdneo** [HubbellCorp/SweetBlue#36] [Q&A]
Hello,
i already sent a mail about the topic some weeks ago but got no response.
So i start a discussion here.

I already update to the new 4.0.1.2-SNAPSHOT version but i am unable to bond new devices.
If you update the app with the version and had already a bonded device this works fine. 

However the android logs shows this :
` onNativeBondFailed() - Bond failed with failReason of BOND_FAIL_REASON_NOT_APPLICABLE(-1). Retrying bond...`

There is also a issue (see [https://github.com/HubbellCorp/SweetBlue/issues/34](https://github.com/HubbellCorp/SweetBlue/issues/34) ) from another guy who also have the same issue. 

Its important to state out that new app and app updates will be impossible after 31th August 2024 if you doesnt update to target sdk 34 (same procedure as every year - see this https://developer.android.com/google/play/requirements/target-sdk).

**Comment by ryanhubbell** [comment]
Version 4.0.1.3-SNAPSHOT has been deployed, which should resolve this issue. Please try it out, and let us know how it went. Once I get some feedback that things are working properly, then we'll push out a production version.

**Comment by neowhoru** [comment]
Hello ryan,
i tried it out and it works now! Thank you for accepting the PR.
